src/org/scratchOrgSettingsGenerator.ts

```diff
@@ -30,23 +29,56 @@ export enum RequestStatus {
 
 const breakPolling = ['Succeeded', 'SucceededPartial', 'Failed', 'Canceled'];
 
-interface ObjectToBusinessProcessPicklist {
-  [key: string]: {
-    fullName: string;
-    default?: boolean;
+export interface SettingType {
+  members: string[];
+  name: string;
+}
+
+export interface PackageFile {
+  '@': {
+    xmlns: string;
   };
+  types: SettingType[];
+  version: string;
 }
 
-export interface BusinessProcessFileContent extends JsonMap {
-  fullName: string;
-  isActive: boolean;
-  values: [
-    {
-      fullName: string;
-      default?: boolean;
+export const writePackageFile = ({
```

Comment:
2 comments:

this method doesn't write anything, it just returns a JSON object representation right, so it should probably be `create...` not `write...`

NIT: it's actually creating a manifest, which is commonly a `package.xml`, might want to rename it to `createManifest` or something more appropriate